src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+
+## Spider Clan Code
+
+All Space Ninjas will now have a Spider Clan Code they will have to live by, and this thus affects gameplay substantially by imposing certain restrictions.
+
+- The already existing one, being the inability to shoot ranged weapons, will be included in this Code.
+- The second restriction is making it so the Space Ninjas can no longer tend to their own wounds - This includes applying topicals, injecting syringes, consuming pills, and drinking liquid chemicals, as it would be preferred that they are not even spotted during their mission. To compensate, Space Ninjas will now have increased passive regeneration as well as faster bloodloss recovery that far exceeds what normal crewmates could ever achieve due to their bio-engineered bodies.
+
+# New Items & Abilities
+
+New content that revolves around both a higher quality of life and a more immersive playfield.
+
+## Flavour Additions
+
+- The Space Ninja now comes with an adequately themed satchel, as well as a katana sheath.
+- The Space Ninja's pinpointer will now act as software for the brand new space ninja PDA instead of being its own gadget, the agent ID card will also be within the aforementioned device.
+- Equipping the space ninja helmet is now an instantaneous toggle action that comes with the space ninja suit.
+
+## Ability Additions
+
+- The Space Ninja can now fabricate ninja caltrops, a variant of ninja throwing stars with an emphasis on piercing damage and bleeding, once deployed, that is, dropped on the ground, they will progressively get more and more camouflaged to the point of near-invisibility. Upon being stepped on, the caltrops will stick to the victim and perform continous bloodloss damage until taken off.
+- The ninja visor can now alternate between normal vision and night vision.
+
+# New Objectives & Mechanics
+
+Gives the Space Ninja more to do and tinker with, so that all rounds don't play out the same.
+
+## Objective Additions
+
+- Steal X spesos - A strongbox for Cargo containing all their deposited spesos will be located in the vault, the Space Ninja is tasked with stealing a designated amount - An alternative to "Steal X technologies".
+- Upload the law board - An AI law board with a Spider Clan-oriented law set, the Space Ninja is tasked with uploading it to the AI upload console - An alternative to "Detonate your spider charge".
+- Hack Si-XXXX - The Space Ninja's kill-target equivalent, they are tasked with hacking a specific cyborg, turning it into a zealous Spider Clan tech-priest that obsesses over hoarding all manners of electronics and machines.
```

Comment:
This is good. The SS13 implementation was transforming a Borg into some psychotic kill machine, but having a Borg that behaves similar to a ninja would be cool.

A suggestion to this point: Borgs should be able to hack the machines the same way a ninja would, albeit at a slower rate, and when completed, they count for the ninja. Might be something interesting to see.

Reply:
> A suggestion to this point: Borgs should be able to hack the machines the same way a ninja would, albeit at a slower rate, and when completed, they count for the ninja. Might be something interesting to see.

I like this idea a lot, my only issue with it is that it''d probably be _way_ too good, I mean, an AA buddy that's not immediately valid? sounds like a dream! maybe the borg itself could get objectives that don't conflict with the ninja's by serving as a distraction? or maybe a "seppuku" module similar to the martyrs that the ninja can activate at will? open for more suggestions